### ISSUE TITLE
message_unsend event never appears

### ISSUE THREAD

**Issue by PixelHir** [Schmavery/facebook-chat-api#718]
Even though I have listenEvents: true, I receive every event except the unsend.
I've put a console.log(event) and nothing happens when removing a message.

**Comment by lequanglam** [comment]
Maybe Facebook changed that event, so the code I wrote about that event didn't work anymore. I will check it and fix the code if necessary.

**Comment by lequanglam** [comment]
![image](https://user-images.githubusercontent.com/29724654/55381820-3e260580-554e-11e9-8612-c6a44f6621df.png)
I can still get the unsend event with current GitHub code. Maybe you want to try this command: `npm i Schmavery/facebook-chat-api` in your folder contains your code.

**Comment by PixelHir** [comment]
Nope, I was already on a newest version.
https://i.imgur.com/o5DPjVk.gifv

**Comment by lequanglam** [comment]
Hmm... can you show me your full code?

**Comment by PixelHir** [comment]
> Hmm... can you show me your full code?

https://github.com/PixelHir/jan-zamachowski
bot.js, the only difference is that there is no console.log

**Comment by lequanglam** [comment]
I see.
Well, unsend event have its own event type ("message_unsend"). You want to add that event type in switch..case too.

**Comment by PixelHir** [comment]
Yeah, however the console.log was added before switch to list all events for debug purposes, it isnt in the code as I said above

**Comment by PixelHir** [comment]
tested this once again, this time with starting from scratch, used code below:

`const fb = require('facebook-chat-api');

fb({email: '---', password: '---'}, (err, api) => {
    api.listen((err, msg) => {
        console.log(msg.type);
    });
});`

**Comment by Hongarc** [comment]
I think `message_unsend` can't send to another user that you don't read it, It just marks in your browser

**Comment by PixelHir** [comment]
The only thing I can think of is that message_unsend is handled in a different way on some accounts.

**Comment by lequanglam** [comment]
Maybe you want to try on a different account.
